invokeai/app/services/processor.py

```diff
@@ -34,8 +36,14 @@ def __process(self, stop_event: Event):
         try:
             self.__threadLimit.acquire()
             while not stop_event.is_set():
-                queue_item: InvocationQueueItem = self.__invoker.services.queue.get()
+                try:
+                    queue_item: InvocationQueueItem = self.__invoker.services.queue.get()
+                except Exception as e:
+                    logger.debug("Exception while getting from queue: %s" % e)
+
                 if not queue_item:  # Probably stopping
+                    # do not hammer the queue
+                    time.sleep(1)
```

Comment:
1 second might be long for responsiveness. Maybe a shorter period of time?

Reply:
I reduced it to 0.5s. There doesn't seem to be a perceptible difference, thanks to the invocation progress bar going into "Preparing" mode immediately after hitting "Invoke".